src/compiler/emitter.ts

```diff
@@ -4766,9 +4766,14 @@ const _super = (function (geti, seti) {
                 emitToken(SyntaxKind.CloseBraceToken, body.statements.end);
             }
 
-            function findInitialSuperCall(ctor: ConstructorDeclaration): ExpressionStatement {
+            /**
+             * Return the statement at a given index if it is a super-call statement
+             * @param ctor constructor declaration
+             * @param index an index to constructor's body to check
+             */
+            function getSuperCallAtGivenIndex(ctor: ConstructorDeclaration, index: number): ExpressionStatement {

```

Comment:
We can't just return boolean because we should the super-call found here for emitting later